Wafa_Alturaifi/week_01/day_02/if-statment/main.js

```diff
@@ -0,0 +1,49 @@
+//var x;
+//var y;
+//var married = z;
+//var kids= n;
+
+//Age Calculator
+var currentyear = 2018;
+var  birthyear = 1969;
+var age = currentyear - birthyear;
+if(age << 40)
+{
+    console.log("your age is"+ age );
+} else {
+console.log("your are too young");
+}
+
+//Supply Calculator
+var currentage = 34;
+var maximumage =75;
+var estimatedamount;
+function etim(currentage,maximumage){
+   estimatedamount=(maximumage-currentage)*2
+   if(estimatedamount=>40){
+    console.log("you have alot time to eat");}
+    else{
+    console.log("harry up no theres no time to eat");}
+
```

Comment:
Nice work here, one thing to note is that we always try to make our code more readable, so for example, we try to have closing curly brackets `}` on their own line to make sure the reader notices them. Let's look at the next code snippets

```
if(estimatedamount=>40){
    console.log("you have alot time to eat");}
    else{
    console.log("harry up no theres no time to eat");}
```

```
if(estimatedamount=>40){
    console.log("you have alot time to eat");
} else {
    console.log("harry up no theres no time to eat");
}
```

Which one do you find more readable?